src/devices/Mpu9250/AccelerometerBandwidth.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Mpu9250
+{
+        /// <summary>
```

Comment:
nit: indent is off

Reply:
hum, that's weird, it doesn't show in VS. Corrected.